prow/cmd/deck/BUILD.bazel

```diff
@@ -11,7 +11,7 @@ filegroup(
 
 container_image(
     name = "asset-base",
-    base = "@alpine-base//image",
+    base = "@git-base//image",
```

Comment:
I'm deeply concerned with the amount of complexity creeping into deck. Sounds like a security nightmare. Why does deck now need a git client?